pkg/scheduler/core/equivalence_cache_test.go

```diff
@@ -692,13 +746,15 @@ func TestInvalidateCachedPredicateItemOfAllNodes(t *testing.T) {
 
 	// there should be no cached predicate any more
 	for _, test := range tests {
```

Comment:
Taking a closer look at this test, I don't like that the various test cases seem to leak state into each other (i.e. loop over the tests twice with the same ecache.)

It's definitely beyond the scope of this PR, but I just wanted to point it out. I should file a separate issue.